scrapeJEmails-NoTechOrBlog.py

```diff
@@ -0,0 +1,129 @@
+import os 
+import re
+from bs4 import BeautifulSoup
+import sys
+
+
+def scrapeEmailsFromSinglePage(linkToScrape): 
+
+	#create progress dots
+	print('.'),
+	sys.stdout.flush()
+	
+	#find current domain in case hrefs linked to straight directories (e.g. href="/apple.html")
+	currentDomain = linkToScrape.replace("http://","").replace("www.","").replace("https://","")
+
+
+	if len(currentDomain.split("/"))>0: 
+		currentDomain = currentDomain.split("/")[0]
+	if "http://" in linkToScrape:
+		currentDomain = "http://"+currentDomain
+	elif "https://" in linkToScrape:
+		currentDomain = "https://"+currentDomain
+	else: 
+		currentDomain = "http://"+currentDomain
+
+	#get the html from the page
+	os.system("./phantomjs myScraper.js "+linkToScrape)
+	currentPage = open("tempPage.txt",'r')
```

Comment:
if we use a context manager, we can be sure the file handle is closed.  something like:

```
with open('tempPage.txt', 'r') as currentPage:
   # your code here
```